src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -818,6 +827,12 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		this.titleControl.updateEditorLabel(editor);
 	}
 
+	private onDidChangeEditorSelection(editor: EditorInput): void {
```

Comment:
Is this correct? When an event is emitted for an editor selection, we seem to set that selection to the title control, but what if other editors are still selected? I was expecting to only change the selection state for the one editor in the event, but not others?